### PR TITLE
Flake8: remove unused imports

### PR DESCRIPTION
**Pull request by squeaky-pl**

### CODE REVIEW

File: tests/auth/test_generic_auth.py [hunk 1]
```diff
@@ -1,12 +1,11 @@
 # -*- coding: utf-8 -*-
-import copy
```
Comment (squeaky-pl) [inline tests/auth/test_generic_auth.py:2]: This is actually unused stdlib import I missed before

**Review by AlecRosenbaum — COMMENTED** [review]
Is the failing test related at all?

**Comment by squeaky-pl** [comment]
Need to investigate this, it seems to be consistently failing in CircleCI but not locally.

File: tests/contacts/test_remote_sync.py [hunk 2]
```diff
@@ -97,7 +97,7 @@ def test_auth_error_handling(contact_sync, default_account, db):
     db.session.commit()
 
     contact_sync.start()
-    contact_sync.join(timeout=10)
+    contact_sync.join(timeout=15)
```
Comment (AlecRosenbaum) [inline tests/contacts/test_remote_sync.py:100]: it this needed still? It looks like the test would be skipped

**Review by AlecRosenbaum — APPROVED** [review]

File: tests/contacts/test_remote_sync.py (hunk 2 above)
Comment (squeaky-pl) [reply tests/contacts/test_remote_sync.py]: Right, I added a separate issue about [removing it](https://github.com/closeio/sync-engine/issues/316).

**Comment by squeaky-pl** [comment]
This triggered following error in `delete-accounts` pod.

Screenshot 2022-02-16 at 09 29 13


All the other pods seemed to be working correctly. Need to figure out which import has side effects of fully initializing SQLAlchemy in delete-accounts pod.

**Comment by squeaky-pl** [comment]
This has rotten now, will open another one.

**Pull request closed without merging**